sdk/Sdk.Generators/FunctionMetadataProviderGenerator/FunctionMetadataProviderGenerator.Parser.cs

```diff
@@ -519,6 +520,21 @@ private bool TryGetReturnTypePropertyBindings(ITypeSymbol returnTypeSymbol, bool
                 return true;
             }
 
+            private bool HasHttpResponseattribute(IPropertySymbol prop)
+            {
+                var attributes = prop.GetAttributes();
+                if (attributes.Length == 1)
```

Comment:
What if customer code has some extra attributes on this property? We should not assume it has only 1.